app/services/commitments/minimum/calculate_true_up_fee_service.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Commitments
+  module Minimum
+    class CalculateTrueUpFeeService < BaseService
+      def initialize(invoice_subscription:)
+        @invoice_subscription = invoice_subscription
+        @minimum_commitment = invoice_subscription.subscription.plan.minimum_commitment
+
+        super
+      end
+
+      def call
+        result.amount_cents = amount_cents
+        result
+      end
+
+      private
+
+      attr_reader :minimum_commitment, :invoice_subscription
+
+      delegate :subscription, to: :invoice_subscription
+
+      def amount_cents
+        return 0 if !minimum_commitment || fees_total_amount_cents >= commitment_amount_cents
+
+        commitment_amount_cents - fees_total_amount_cents
+      end
+
+  		def commitment_amount_cents
```

Comment:
Layout/IndentationWidth: Use 2 (not 0) spaces for indentation. (https://rubystyle.guide#spaces-indentation)